src/project/manifest/mod.rs

```diff
@@ -308,15 +308,13 @@ impl Manifest {
         dep: &PackageName,
         spec_type: SpecType,
         platform: Option<Platform>,
+        feature_name: Option<&FeatureName>,
```

Comment:
I think you can use the same logic here. I think the user should always pass a feature name. Its either the default or a named one.

Reply:
Ah yeah, changed that in d8dbc95